### PR TITLE
feat:added grid documentation

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/154864975-0b96459b-97ee-4bfc-b872-cbb576bc8580.mp4

### CODE REVIEW

File: components/grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grids||GALAXIA</title>
+</head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="../../docs/components.css">
+<link rel="stylesheet" href="../../landing page/style.css">
+<link rel="stylesheet" href="grids.css">
+<script src="https://kit.fontawesome.com/17540dc721.js" crossorigin="anonymous"></script>>
```
Comment (nareshpoonia) [inline components/grids/grids.html:16]: You can consider putting the font-awesome script in Main CSS, so that you don't have to add it to all of your HTML

**Review by shuklahimanshu81 — APPROVED** [review]
Liked the clean code structure. Good to go for merge.

**Review by nareshpoonia — APPROVED** [review]
Code looks neat & clean, just one comment on font-awesome script 
Good to merge

File: components/grids/grids.html (hunk 1 above)
Comment (stutisk) [reply components/grids/grids.html:16]: you are right will make the changes

**Pull request merged**